roles/network_plugin/cilium/templates/cilium-config.yml.j2

```diff
@@ -0,0 +1,29 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cilium-config
+  namespace: {{ system_namespace }}
+data:
+  # This etcd-config contains the etcd endpoints of your cluster. If you use
+  # TLS please make sure you uncomment the ca-file line and add the respective
+  # certificate has a k8s secret, see explanation bellow in the comment labeled
+  # "ETCD-CERT"
+  etcd-config: |-
+    ---
+    endpoints: 
+{% for ip_addr in etcd_access_addresses.split(',') %}
```

Comment:
So it stores data in the kube-system etcd, right? AFAIK, that is a major security concern from k8s architecture best practices perspective. Could you please note that in the docs in follow up patches?